File: AdminService/src/main/java/com/ecommercial/shopping/adminservice/admin/domain/vo/Address.java
package com.ecommercial.shopping.adminservice.admin.domain.vo;

import jakarta.persistence.Embeddable;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Embeddable
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Address {
    private String city;
    private String firstAddress;
    private String secondAddress;
    private String zipCode;
}

File: UserService/src/main/kotlin/com/ecommercial/shopping/userservice/user/domain/vo/Address.kt
package com.ecommercial.shopping.userservice.user.domain.vo

import jakarta.persistence.Embeddable
import lombok.NoArgsConstructor

@Embeddable
data class Address(
    val city: String = "",
    val firstAddress: String = "",
    val secondAddress: String = "",
    val zipCode: String = ""
) {
}